src/network/networkd-network.h

```diff
@@ -102,7 +102,7 @@ struct Network {
         int allmulticast;
         int promiscuous;
         bool unmanaged;
-        bool required_for_online; /* Is this network required to be considered online? */
+        int required_for_online; /* Is this network required to be considered online? */
```

Comment:
Please change conf parser to config_parse_tristate().

Reply:
i don't think that would work, as the value can also be an operstate range; i think we need to keep using the config_parse_required_for_online() function, right?

Reply:
Indeed. Sorry for the noise.